src/js/_enqueues/wp/webp-fallback.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * @output wp-includes/js/webp-fallback.js
+ */
+
+window.media = window.media || {};
```

Comment:
Why put this in a `media` global? For WordPress core, `wp` is commonly used, so let's use that here too I would suggest.